problem8.py

```diff
@@ -0,0 +1,56 @@
+#Find the thirteen adjacent digits in the 1000-digit number that have the greatest product.
+def main():
+    long_number = ("73167176531330624919225119674426574742355349194934"+
```

Comment:
Did you test it after adding the brackets?

Reply:
Yes, it works the same as "\", but better ;)